geode-core/src/main/java/org/apache/geode/internal/cache/versions/RegionVersionVector.java

```diff
@@ -113,7 +113,7 @@
   private transient boolean isLiveVector;
 
   /** for debugging we keep a reference to the region for use in log messages */
```

Comment:
This class now also uses the "region" field to check if the region is initialized so this comment is out of date.
I recommend removing the comment on this field.

Reply:
New changes posted by removing comments.